packages/vuetify/src/util/mergeData.ts

```diff
@@ -155,3 +112,43 @@ export default function mergeData (): VNodeData {
 
   return mergeTarget
 }
+
+export function mergeStyles (target: undefined | string | object[] | object, source: undefined | string | object[] | object) {
```

Comment:
break into new lines like this https://github.com/vuetifyjs/vuetify/pull/11271/files?diff=split&w=1#diff-1dc448fb0b8090117cb2dca3374a0542R133